cfme/tests/openstack/infrastructure/test_provider.py

```diff
@@ -15,8 +15,8 @@
 def test_api_port(provider):
     view_details = navigate_to(provider, 'Details')
     port = provider.data['endpoints']['default']['api_port']
-    assert int(view_details.entities.properties.get_text_of('API Port')) == port, 'Invalid API Port'
-
+    assert int(view_details.entities.summary('Properties').get_text_of('API Port')) == port, \
```

Comment:
REQUIRED: according to our guidelines we use () instead of \. Please fix this and I'll merge your PR

Reply:
Hi,

Use () instead of what ?. Did not get you requirement

Reply:
You mean to enclose the whole expression in () in order to break line instead of \